token/services/selector/config/driver.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config
+
+import (
+	"time"
+
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/selector/driver"
+	"github.com/pkg/errors"
+)
+
+const (
+	defaultDriver            = driver.Sherdlock
+	defaultEvictionInterval  = 3 * time.Minute
```

Comment:
The defaults were:
- cleanupPeriod (evictionInterval): 30 seconds
- cleanupTickPeriod : 1 minute.

I don't know what good defaults would be, maybe something inbetween? We want to give some space for exceptionally slow commits, but not lock tokens for too long if something went wrong... Tick every 90 seconds, clean every lock older than a minute?

Reply:
yeah, I also don't know. I would leave evictionInternal to 3 minutes and the tick to 90 seconds.

do you think that evictionInternal is a good name? What about lockAge?

Reply:
leaseExpiry?

Reply:
perfect